projects/laji/src/app/shared-modules/spreadsheet/service/spreadsheet.service.ts

```diff
@@ -289,18 +303,46 @@ export class SpreadsheetService {
     return `${('' + N).replace(',', '.')},${('' + E).replace(',', '.')}${suffix}`;
   }
 
-  private getCombinedGroups(combines: IColCombine[]): IColCombine[][] {
+  private getCombinedGroups(combines: IColCombine[]): {groupedCombines: IColCombine[][], errors: string[]} {
+    const errors = [];
+    combines = this.checkCombinedHasRequiredColumns(combines, GroupTypeEnum.date, errors);
+    combines = this.checkCombinedHasRequiredColumns(combines, GroupTypeEnum.coordinate, errors);
+
     const groups: {[key: string]: IColCombine[]} = {};
     combines.forEach(col => {
       if (!groups[col.groupId]) {
         groups[col.groupId] = [];
       }
       groups[col.groupId].push(col);
     });
-    return Object.keys(groups).map(group => {
-      groups[group].sort((a, b) => a.order - b.order);
-      return groups[group];
-    });
+    return {
+      groupedCombines: Object.keys(groups).map(group => {
+        groups[group].sort((a, b) => a.order - b.order);
+        return groups[group];
+      }),
+      errors
+    };
+  }
+
+  private checkCombinedHasRequiredColumns(combines: IColCombine[], groupType: GroupTypeEnum, errors: string[]): IColCombine[] {
+    const requiredKeys = (groupType === GroupTypeEnum.date ? [GeneratorService.splitDate.dd, GeneratorService.splitDate.mm, GeneratorService.splitDate.yyyy] :
+      [GeneratorService.splitCoordinate.E, GeneratorService.splitCoordinate.N, GeneratorService.splitCoordinate.system]
+    );
```

Comment:
Quite long line, could the ternary operation be splitted to multiple lines like this (IMO the most readable option anyway)

```
question
  ? yes
  : no
```

We should add a max line length rule to linting...